xinference/api/restful_api.py

```diff
@@ -1330,8 +1330,10 @@ async def create_embedding(self, request: Request) -> Response:
             raise HTTPException(status_code=500, detail=str(e))
 
         try:
-            embedding = await model.create_embedding(body.input, **kwargs)
-            return Response(embedding, media_type="application/json")
+            embedding_bytes = await model.create_embedding(body.input, **kwargs)
+            embedding = json.loads(embedding_bytes.decode("utf-8"))
+            embedding["model"] = model_uid  # type: ignore
```

Comment:
Try to pass original model uid into model, instead of parsing, adding info then serializing, this is too ugly.